src/index.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+var Image = require('image-js');
+
+const defaultOptions = {
+    lowThreshold: 10,
+    highThreshold: 30,
+    blur: 1.1,
+    brightness: 255
```

Comment:
It's just that the values here are only valid for 8-bit images. Instead, they should be calculated based on the `image.maxValue` property of the image. It would be 255 for 8-bit images and 65535 for 16-bit images.